channels/bo.m3u

```diff
@@ -5,16 +5,22 @@ https://593b04c4c5670.streamlock.net/8192/8192/playlist.m3u8
 http://186.121.206.197/live/daniel/index.m3u8
 #EXTINF:-1 tvg-id="ATB.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/oszItYJ.jpg" group-title="",ATB (360p) [Not 24/7]
 https://mediacp.hostradios.com.ar:19360/atbcochabamba/atbcochabamba.m3u8
-#EXTINF:-1 tvg-id="BoliviaTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="http://www.boliviatv.bo/principal/assets/images/the-voyager/btv1.png" group-title="",Bolivia TV (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="BoliviaTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="http://www.boliviatv.bo/principal/assets/images/the-voyager/btv1.png" group-title="General",Bolivia TV (720p) [Not 24/7]
 http://boliviatv1.srfms.com:5735/live/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="",Bolivia TV 7.2 (720p)
+#EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="General",Bolivia TV 7.2 (720p)
 https://video1.getstreamhosting.com:1936/8224/8224/playlist.m3u8
+#EXTINF:-1 tvg-id="CTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="" group-title="Local",CTV Canal 38 Digital (Viacha)
+https://media2.cdnlayer.biz:8081/8026/index.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/141065030-ab793316-a564-42a1-924d-178826058dc1.png)

Reply:
Tagged as Not 24/7